### ISSUE TITLE
Have Buffer List highlight the previous buffer by default

### ISSUE THREAD

**Issue by jwmann** [vim-ctrlspace/vim-ctrlspace#175]
This is a feature I used to much effect from CtrlP.
I bound the buffer list to `<CR>` so I can effectively press `<CR><CR>` to go to the previous buffer.

In CtrlSpace the default seems to highlight your current buffer, which is essentially useless.

Is there anyway to get the mentioned behaviour?

Thank you.

**Comment by kswope** [comment]
I have to agree with the original poster, that's why I ended up here.

Opening a buffer list, defaulted to the previous buffer, is pretty much a standard these days.  Ctrlp does it, vscode does it, sublime text does it and I think textmate does it.

If you think about it, defaulting to the current buffer **offers zero value**, but selecting the previous buffer offers great value, because it allows the toggling between two buffers, which is commonly needed behavior for development.

**Comment by cometsong** [comment]
By this do you mean that opening a buffer list and having the previous buffer selected by default is an easier way to switch between them than using `<CTRL-^>` to swap buffers in current window, or `<CTRL-w>p` (or`<CTRL-w><CTRL-p>`) to switch to most-recent previously-used window?

**Comment by kswope** [comment]
> By this do you mean that opening a buffer list and having the previous buffer selected by default is an easier way to switch between them than using <CTRL-^> to swap buffers in current window, or <CTRL-w>p (or<CTRL-w><CTRL-p>) to switch to most-recent previously-used window?

Honestly can't tell if you are being sarcastic.  

Whats the point of defaulting to the current buffer?  It offers no value.

Whats the point of making users remember another key combo? 

Why does every other editor that uses the "quick open" feature doing this?

Ctrl-^ is difficult to type for some users, I know it is for me.  The other options are worse.  Since I switched over from Ctrlp I've had to remap Ctrl-^ because ctrlp made ctrl-^ obsolete.

**Comment by cometsong** [comment]
Nope, not sarcasm, just personal usage differences.  Super sorry it seemed sarcastic!

I personally disagree that the previous buffer being the default selected is useless, as it doesn't fit into my own workflows. For instance there are times when I open the CtrlSpace window and use it to move the current buffer to a new tab using `t` and work on it there.  To switch to previous buffer, I just use the techniques I mentioned before (mostly from muscle habits prior to using any file/buffer/workspace-manager). 

One way the task described by the initial post  _can_ be done is open the CtrlSpace window and type capital `P` which switches to the previously used item and opens it.  True it's an extra key to press, but it gets the job done. This can be mapped to your own choice of keypresses.
e.g. `map <cr><cr> :CtrlSpace P<CR>`

_(As a ide note:_`<C-^>` _task can also be done with_ `<C-6>`_to avoid the shift key.)_

**Comment by kswope** [comment]
Maybe something like this would be a simple compromise

```
:CtrlSpaceP
```
